run.sh

```diff
@@ -47,10 +61,26 @@ if [ "$MKV_GID" -ne "0" ] && [ "$MKV_GID" -ne "$(id -g mkv)" ]; then
     groupmod -og "$MKV_GID" mkv
 fi
 
+# add the "mkv" user to a group that can work on the cdrom
+device_group=$(stat -c "%G" "$device")
+if [ "$device_group" = "UNKNOWN" ]; then
+    device_gid=$(stat -c "%g" "$device")
+    device_group=cdrom_docker
+
+    groupadd --gid "$device_gid" "$device_group"
+fi
+usermod -a -G "$device_group" mkv
+
 # Determine destination directory.
 ID_FS_LABEL=${ID_FS_LABEL:-$(blkid -o value -s LABEL)}
+echo "ID_FS_LABEL=$ID_FS_LABEL"
 ID_FS_UUID=${ID_FS_UUID:-$(blkid -o value -s UUID)}
+echo "ID_FS_UUID=$ID_FS_UUID"
+
 TEMPLATE="${ID_FS_LABEL:-nolabel}_${ID_FS_UUID:-nouuid}_XXX"
+echo "TEMPLATE=$TEMPLATE"
+
+chown mkv:mkv /output
```

Comment:
What was your docker run command? I left this owned by non-mkv to prevent makemkvcon from creating huge containers when the user accidentally forgets to bind-mount /output.